Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs

```diff
@@ -17,7 +17,7 @@ internal static class IndexSearch
     {
 
         // Reserved keywords in oleDB
-        private const string reservedStringPattern = @"^[`\@\#\^,\&\/\\\$\%_;\[\]]+$";
+        private const string reservedStringPattern = @"^[`\@\＠\#\＃\＊\^,\&\＆\/\\\$\%_;\[\]]+$";
```

Comment:
I am curious why these additional character need escape? They are not really special character for regex.

Reply:
if those characters are entered alone, an error in "db".
Its so weird..

Reply:
In Korean, special characters are sometimes entered with function keys. There is a problem that is recognized the same even if # and # are different.